kyuubi-server/src/test/scala/org/apache/kyuubi/credentials/HiveDelegationTokenProviderSuite.scala

```diff
@@ -118,7 +120,7 @@ class HiveDelegationTokenProviderSuite extends KerberizedTestHelper {
       assert(aliasAndToken._2 != null)
 
       val token = aliasAndToken._2
-      val tokenIdent = token.decodeIdentifier().asInstanceOf[DelegationTokenIdentifier]
+      val tokenIdent = token.decodeIdentifier().asInstanceOf[AbstractDelegationTokenIdentifier]
```

Comment:
we still have hive dependencies for tests, why do we need change this ?

Reply:
actually, we should use the shaded `DelegationTokenIdentifier` instead

Reply:
BTW, the test scope hive jars are going to be removed soon